### PR TITLE
Admin grammar fixes and improvements

### PR DESCRIPTION
**Pull request by pgaskin**
Admin grammar fixes and improvements.

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1138,17 +1138,17 @@ users.update_profile_success = Account profile has been updated.
...
-users.is_admin = This account has administrator permissions
-users.allow_git_hook = This account has permission to create Git hooks
-users.allow_import_local = This account has permissions to import local repositories
-users.allow_create_organization = This account has permissions to create organizations
+users.is_activated =  Account activated
+users.prohibit_login = Login disabled
+users.is_admin = Administrator permissions
+users.allow_git_hook = Allowed to create git hooks
+users.allow_import_local = Allowed to import local repositories
+users.allow_create_organization = Allowed to create organizations
```
Comment (pgaskin) [inline options/locale/locale_en-US.ini:1245]: The above ones are checkboxes on the user edit page, so no need to say `This account`. This is why I changed these lines.

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -1138,17 +1138,17 @@ users.update_profile_success = Account profile has been updated.
...
 users.update_profile = Update Account Profile
-users.delete_account = Delete This Account
-users.still_own_repo = This account still owns at least one repository, you need to delete or transfer them first.
-users.still_has_org = This account still is a member of at least one organization, you need to leave or delete the organizations first.
-users.deletion_success = Account has been deleted successfully.
+users.delete_account = Delete Account
+users.still_own_repo = This user still owns one or more repositories. These repositories need to be deleted or transfered first.
```
Comment (cez81) [inline options/locale/locale_en-US.ini]: transferred

Comment (pgaskin) [reply options/locale/locale_en-US.ini]: Thanks

**Comment by cez81** [comment]
LGTM

File: options/locale/locale_en-US.ini [hunk 3]
```diff
@@ -1080,8 +1080,8 @@ dashboard.delete_missing_repos = Delete all repository records which have lost t
 dashboard.delete_missing_repos_success = All repository records which have lost their corresponding Git files have been deleted.
 dashboard.git_gc_repos = Execute garbage collection on all repositories
 dashboard.git_gc_repos_success = All repositories have finished executing garbage collection.
-dashboard.resync_all_sshkeys = Rewrite '.ssh/authorized_keys' file (for Gitea SSH keys)
-dashboard.resync_all_sshkeys_success = All public keys have been rewritten.
+dashboard.resync_all_sshkeys = Rewrite '.ssh/authorized_keys' file (for Gitea SSH keys). Do not use this if you are using the built-in SSH server.
```
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: This should be `Disabled if you are using the built-in SSH server.` no? As in it won't do anything...

File: options/locale/locale_en-US.ini [hunk 4]
```diff
@@ -1080,8 +1080,8 @@ dashboard.delete_missing_repos = Delete all repository records which have lost t
...
+dashboard.resync_all_sshkeys_success = All Gitea public keys have been rewritten.
```
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: `All public keys controlled by Gitea [...]`

File: options/locale/locale_en-US.ini [hunk 5]
```diff
@@ -1138,17 +1138,17 @@ users.update_profile_success = Account profile has been updated.
...
+users.allow_import_local = Allowed to import local repositories
+users.allow_create_organization = Allowed to create organizations
 users.update_profile = Update Account Profile
-users.delete_account = Delete This Account
-users.still_own_repo = This account still owns at least one repository, you need to delete or transfer them first.
-users.still_has_org = This account still is a member of at least one organization, you need to leave or delete the organizations first.
-users.deletion_success = Account has been deleted successfully.
+users.delete_account = Delete Account
+users.still_own_repo = This user still owns one or more repositories. These repositories need to be deleted or transferred first.
+users.still_has_org = This user is still is a member of one or more organizations. This user needs to leave or delete them first.
```
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: Is this true? Is it not only if you're an Owner of an Org?

File: options/locale/locale_en-US.ini [hunk 6]
```diff
@@ -1180,7 +1180,7 @@ auths.host = Host
 auths.port = Port
 auths.bind_dn = Bind DN
 auths.bind_password = Bind Password
-auths.bind_password_helper = Warning: This password is stored in plain text. Do not use a highly privileged account.
+auths.bind_password_helper = Warning: This password is stored in plain text. Do not use a high privilege account.
```
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: Maybe `Only use a read-only account` ?

**Review by bkcsoft — COMMENTED** [review]
I know it's just a grammar-check, but we might as well touch some of them up while we're at it 😉

**Comment by andreynering** [comment]
LGTM

**Comment by lafriks** [comment]
Please do not merge this before 1.2.0 is released as this will break already translated strings

**Review by bkcsoft — CHANGES_REQUESTED** [review]
@lafriks You can do empty "request changes" to "stall" PRs ;)

**Comment by lafriks** [comment]
@geek1011 rebase

**Comment by lunny** [comment]
Any update? Or we should close this one?

**Comment by pgaskin** [comment]
Sorry, I have been quite busy for the last few weeks. I will update it soon.

**Comment by pgaskin** [comment]
@lunny @bkcsoft 

Also, sorry for the delay, I have been really busy recently.

**Comment by lafriks** [comment]
@bkcsoft need your approval

**Comment by lunny** [comment]
@lafriks CI failed

**Comment by lunny** [comment]
need @bkcsoft approval

**Review by bkcsoft — APPROVED** [review]

**Comment by lafriks** [comment]
Make LG-TM work again

**Pull request merged**